infra/fastly/terraform/dl.k8s.io/vcl/binaries.vcl

```diff
@@ -0,0 +1,144 @@
+sub vcl_recv {
+  # configure purges to require api authentication:
+  # https://docs.fastly.com/en/guides/authenticating-api-purge-requests
+  #
+  if (req.method == "FASTLYPURGE") {
+      set req.http.Fastly-Purge-Requires-Auth = "1";
+  }
+
+  # Prevent edge from caching stale content served from shield
+  # https://developer.fastly.com/learning/concepts/stale/#shielding-considerations
+  if (fastly.ff.visits_this_service != 0) {
+    set req.max_stale_while_revalidate = 0s;
+  }
+
+
+#FASTLY recv
+
+  # don't bother doing a cache lookup for a request type that isn't cacheable
+  if (req.method !~ "(GET|HEAD|FASTLYPURGE)") {
+    return(pass);
+  }
+
+  return(lookup);
+}
+
+sub vcl_fetch {
+
+  if (beresp.status >= 200 && beresp.status < 300) {
+    set beresp.cacheable = true;
```

Comment:
This [isn't needed](https://developer.fastly.com/reference/vcl/variables/backend-response/beresp-cacheable/) as you have `req.enable_segmented_caching` set to true, but no harm in being explicit.